frontend/insurance/src/app/core/components/navigation-bar/navigation-bar.component.scss

```diff
@@ -1,3 +1,4 @@
+//Keeping in case we want to switch to logos later
```

Comment:
make sure to run `ng lint` to make sure there are no errors that show up. this should throw an error to the linter.